sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration/ConfigurationAsyncClientJavaDocCodeSnippets.java

```diff
@@ -12,10 +12,10 @@
  */
 public class ConfigurationAsyncClientJavaDocCodeSnippets {
 
-    private String key1 = "key1";
-    private String key2 = "key2";
-    private String value1 = "val1";
-    private String value2 = "val2";
+    private static final String KEY_1 = "key1";
+    private static final String KEY_2 = "key2";
+    private static final String VALUE_1 = "val1";
+    private static final String VALUE_2 = "val2";
```

Comment:
Not sure why these were changed to static and renamed?

Reply:
This was part of the last Connie's requirement.
We check for cases where a field is initializated and it's value never changes (in constructor or methods) and ask for those field to also be final.
Then, if field private field becomes `final` , spotbug complains and say that it must be static.
Then, when it became `static final`, checkstyle asks to use a valid constant name (uppercase)

so, yeah, I chain of style requeriments :D .  But at the end, it makes sense for these fields to be constants, for the way it is used

Reply:
I think it is a function of the contrived nature of our samples. Please take a look at the code from the perspective of what someone expects to see (when it is presented in the form of a JavaDoc code snippet). If it doesn't make sense for it to be presented as a static final constant, please revert and make it appropriate. Add suppressions when necessary.

Reply:
I see. yeah, that makes sense. This needs a suppression and not a code change! 👍